README.md

```diff
@@ -1,5 +1,45 @@
 # contributr
-An online tool for programmers to find projects they can contribute to and project maintainers to find contributors. 
 
 [![Build Status](https://travis-ci.org/Djenesis/contributr.svg)](https://travis-ci.org/Djenesis/contributr)
+[![Build Status](https://travis-ci.org/Djenesis/contributr.svg?branch=master)](https://travis-ci.org/Djenesis/contributr)
 [![codecov.io](http://codecov.io/github/Djenesis/contributr/coverage.svg?branch=master)](http://codecov.io/github/Djenesis/contributr?branch=master)
+
+An online tool for programmers to find projects they can contribute to and project maintainers to find contributors. 
+
+## What is contributr?
+__contributr__ is an online tool that helps contributors to find projects they can contribute to based on their interests and skills. It can also help projects owners and maintainers find potential contributors.
+
+### how would this work?
+You login with your Github account, __contributr__ finds projects based on your interests and skill levels. You now have a list of projects you can start contributing to!
+
+### when will it be ready?
+When it is ready. It'll take a little time to get started, since it's a learner-group project.
+
+## What is contributr built with?
+`python`!!! Or more specifically, `django`. The whole idea started as a _django_ learning group, and so it's all been built from the bottom up from there.
+
+## Installation
+the usual, 
+
+  - clone the git repo or download a zip
+  - install the dependencies 
+  - run the django project
+
+## How do I contribute?
+> I highly suggest you __fork__ the repo and start from there.
+
+If you want to join the learning group, just `fork` the repo, and follow us on the __slack__ channel.
+
+You can create an __`issue`__ for a `feature` or a `bug` and follow up with a pull-request. IF you're a novice regarding all this, read up Github's excellent documentation.
+
+### Features
+Can be any small little thing that you just learnt about, or something BIG that is complex, but awesome! __contributr__ has space for everything.
+
+### How about just ideas?
+Great! Come join our group-talks at slack and fire away all those awesome ideas in your brain.
+
+### Testing?
+Sure! In fact, the more tests you run, the better chance you have at catching bugs! So you can write tests, welcome! If you can run tests and find bugs, still welcome!
```

Comment:
Add some lines about testing.

The testing dependencies are located at `requirements/testing.txt`

After a pip install from that, one would run `pytest` from that.